src/Mapping/Operator/Simple/Explode.php

```diff
@@ -58,9 +58,15 @@ public function process($inputData, bool $dryRun = false)
 
                 return $explodedArray;
             } else {
+                if (empty($inputData)) {
```

Comment:
we could move that if to the beginning of the `process` method, then we would not need to duplicate it, right?